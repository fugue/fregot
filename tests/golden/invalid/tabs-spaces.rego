# This test really just ensures that we print the "^^^^^" 
package fregot.tests.invalid.tabs_spaces

test_tab {
	1 > "what?"
}

test_spaces {
    1 > "what?"
}

test_mixed_01 {
	 	  1 > "what?"
}
