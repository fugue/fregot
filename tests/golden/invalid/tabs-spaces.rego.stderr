fregot ([34mtypecheck error[0m):
  "tabs-spaces.rego" (line 5, column 9):
  Subtype error:

    5| 	1 > "what?"
       	[31m^^^^^^^^^^^[0m

  Cannot use type [33m[34mstring[0m[0m as [33m[34mnumber[0m[0m

fregot ([34mtypecheck error[0m):
  "tabs-spaces.rego" (line 9, column 5):
  Subtype error:

    9|     1 > "what?"
           [31m^^^^^^^^^^^[0m

  Cannot use type [33m[34mstring[0m[0m as [33m[34mnumber[0m[0m

fregot ([34mtypecheck error[0m):
  "tabs-spaces.rego" (line 13, column 19):
  Subtype error:

    13| 	 	  1 > "what?"
        	 	  [31m^^^^^^^^^^^[0m

  Cannot use type [33m[34mstring[0m[0m as [33m[34mnumber[0m[0m
